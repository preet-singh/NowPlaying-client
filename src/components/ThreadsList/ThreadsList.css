
.threads-list-container {
    /* margin-top: 20px; */
}

.threads-list {
    background-color: rgba(0,0,0,0.6);
    margin-top: 30px;
}
.threads-list > h3 {
    font-size: 32px;
    color: #00d838;
    background-color: black;
}

.threads-list h4 {
    font-style: italic;
    text-decoration: none;
    margin: 10px 0px 0px 0px;
}

.threads-list h4:hover a {
    color: #00d838;
}

.threads-list h4 a {
    text-decoration: none;
}

@media screen and (min-width: 968px) {
    .threads-list {
        margin-left: 10%;
        margin-right: 10%;
    }
}

@media screen and (min-width: 768px){
}