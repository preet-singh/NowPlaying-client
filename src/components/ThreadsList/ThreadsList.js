import React from 'react';
import ThreadItem from '../ThreadItem/ThreadItem';
import './ThreadsList.css';

function ThreadsList(props) {
  return (
    <div className='threads-list'>
      <h3>Movies</h3>
      <ThreadItem />
      <ThreadItem />
      <ThreadItem />
    </div>
  )
}

export default ThreadsList;