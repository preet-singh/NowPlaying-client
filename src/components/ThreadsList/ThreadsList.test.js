import React from 'react';
import ThreadsList from './ThreadsList.js';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`ThreadsList component`, () => {

  it('renders the ThreadsList', () => {
    const wrapper = shallow(<ThreadsList />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})