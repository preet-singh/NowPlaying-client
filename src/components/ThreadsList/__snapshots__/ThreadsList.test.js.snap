// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThreadsList component renders the ThreadsList 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
